docs/riff_streaming_kafka-provider_create.md

```diff
@@ -0,0 +1,43 @@
+---
+id: riff-streaming-kafka-provider-create
+title: "riff streaming kafka-provider create"
+---
+## riff streaming kafka-provider create
+
+create a kafka provider of messages
+
+### Synopsis
+
+<todo>
+
+```
+riff streaming kafka-provider create <name> [flags]
+```
+
+### Examples
+
+```
+riff streaming kafka-provider create my-kafka-provider --bootstrap-servers kafka.local:9022
```

Comment:
i think most of the kafka docs use port 9092 by convention.
e.g. https://kafka.apache.org/quickstart#quickstart_createtopic